docs/docs/forms.md

```diff
@@ -383,6 +383,67 @@ const RequiredField = ({ label, name, validation }) => {
 }
 ```
 
+### Controlled Component Fields
+
+If you're working with a fully-loaded component library, or bring your own production-ready components you may want them to integrate with Redwood seemlessly.
+
+You can create these custom fields through the use of Redwood's `useErrorStyles` and React Hook Form's `Controller`. 
+
+The following example shows a component from [`primereact`](https://www.primefaces.org/primereact/) which can be used in Forms like any of the named input fields (listed above).
+
+```jsx
+import { ToggleButton, ToggleButtonProps } from 'primereact/togglebutton'
+
+import { Controller, RegisterOptions, useErrorStyles } from '@redwoodjs/forms'
+
+interface Prop extends ToggleButtonProps {
+  validation?: RegisterOptions
+  errorClassName?: string
+}
+
+const ToggleButtonField = (prop: Prop) => {
+  const {
+    name,
+    className,
+    errorClassName,
+    defaultValue,
+    validation,
+    style,
+    ...propRest
+  } = prop
```

Comment:
```suggestion
    ...propsRest
  } = props
```

Reply:
Personal preference here, I've just always seen it pluralized

Reply:
I usually argue that the pluralized form should be used if the arguments contain multiple objects to clarify that the props can be destructed multiple times. This is definitely the case here. Thanks for your feedback.